src/test/java/com/gtnewhorizons/angelica/glsm/GLSM_PushPop_UnitTest.java

```diff
@@ -257,7 +257,7 @@ void testPushPopEnableBit() {
         bits.add(new GLBit(GL11.GL_TEXTURE_GEN_R, "Texture Gen R", false));
         bits.add(new GLBit(GL11.GL_TEXTURE_GEN_Q, "Texture Gen Q", false));
 
-        GLStateManager.glPushAttrib(GL11.GL_ENABLE_BIT);
+        GLStateManager.glPushAttrib(GL11.GL_ALL_ATTRIB_BITS);
```

Comment:
This test was _trying_ to test GL_ENABLE_BIT in isolation.  If it's setting other state LMK what and we can manually reset it.